nixos/modules/services/security/fprintd.nix

```diff
@@ -50,6 +50,14 @@ in
 
     systemd.packages = [ cfg.package ];
 
+    # The upstream unit does not use StateDirectory, so will
+    # fail if the directory it needs is not present. Should be
```

Comment:
I'm pretty sure `systemd.services.fprintd.serviceConfig.StateDirectory = "fprint"` works too, should get merged with the upstream unit

Reply:
Huh, does that work even when there's an existing upstream unit that we use?

Reply:
Great, seems to work. I'll use that instead.

Reply:
Yeah I found out about this too recently, seems that systemd can merge multiple unit files at runtime.